invenio_opendata/testsuite/data/cms/cms-vm.xml

```diff
@@ -0,0 +1,97 @@
+<collection xmlns="http://www.loc.gov/MARC21/slim">
+   <record>
+    <controlfield tag="001">252</controlfield>
+    <datafield tag="110" ind1=" " ind2=" ">
+      <subfield code="a">CMS Collaboration</subfield>
+    </datafield>
+    <datafield tag="245" ind1=" " ind2=" ">
+      <subfield code="a">CMS VM Image, for 2011 CMS open data</subfield>
+    </datafield>
+    <datafield tag="260" ind1=" " ind2=" ">
+      <subfield code="b">CERN Open Data Portal</subfield>
+      <subfield code="c">2016</subfield>
+    </datafield>
+    <datafield tag="264" ind1=" " ind2="0">
+      <subfield code="c">2011</subfield>
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">The virtual machine image is based on CernVM (cernvm.cern.ch), with Scientific Linux CERN 6 (slc6), which is the environment needed for the CMSSW release 
+      to be used with the 2011 CMS open data. Access to the data is through XRootD.</subfield>
```

Comment:
Oughtn't we to expand this point about access to the data, given the efforts to host the data on CVMFS?